src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public class Machine {
+    public abstract void doWork() {};
+    public abstract void stopWork() {};
```

Comment:
The methods `doWork` and `stopWork` should be declared as abstract without any implementation. Remove the curly braces and semicolons to properly declare them as abstract methods: `public abstract void doWork();` and `public abstract void stopWork();`.